femmecubator/open-sprints
# Task/create project side bar

#### What does this PR do?
- Configure side bar for Projects page

#### How should this be manually tested?
http://localhost:3000/open-sprints/docs/projects/
- Verify that functionality fulfills criteria in ticket - https://trello.com/c/9e1clawJ/15-us-akita002-navigation-sidebar

#### Any background context you want to provide?
- Active states in local don't match 100% with [mock](https://www.figma.com/file/QlZDY3OZn7u2X0R67V7g31/Project%3A-Open-Sprint?node-id=90%3A2425&mode=dev) as it is difficult to override some of Docusaurus's defaults. Alternative would be to swizzle (aka building the side bar from scratch which would be time-consuming).
